testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Standard.java

```diff
@@ -113,7 +113,7 @@ private DockerContainer createPrestoMaster()
     @SuppressWarnings("resource")
     private DockerContainer createTestsContainer()
     {
-        DockerContainer container = new DockerContainer("ghcr.io/trinodb/testing/centos6-oj8:" + imagesVersion, TESTS)
+        DockerContainer container = new DockerContainer("ghcr.io/trinodb/testing/centos7-oj8:38", TESTS)
```

Comment:
Can we keep `imagesVersion` here?

it will go  out of sync otherwise

Reply:
Yes, I'll revert this change when docker images are buildable and released altogether